homeassistant/components/spotify/media_player.py

```diff
@@ -241,6 +241,16 @@ def media_playlist(self):
             return None
         return self._playlist["name"]
 
+    @property
+    def media_context_id(self):
```

Comment:
There is no such thing as a `media_context_id`. This property isn't used anywhere either.

What should this do? And how have you tested this PR?